data/scopes/defaults.ini

```
@@ -5,13 +5,16 @@ displayName = ""
 
 [data]
 tmpl_phone = "fallback.tmpl"
-hostname =
-provisioning_url_scheme = "http"
+provisioning_url_scheme = "https"
 provisioning_freq = "everyday"
-sip_tls_port = 5061
-sip_udp_port = 5060
-ringtone = 1
+sip_tls_port = "5061"
+sip_udp_port = "5060"
+ringtone = "1"
 timezone = "UTC"
 tonezone = "us"
 language = "en"
 ntp_server = "pool.ntp.org"
+call_waiting_tone = "1"
+date_format = "DD MM YY"
+time_format = "24"
+dss_transfer = "attended"
```

Please add also the defaults for the ldap settings

```suggestion
dss_transfer = "attended"
ldap_tls = "none"
```